### ISSUE TITLE
Placeholder position issue while using multiline?

### ISSUE THREAD

**Issue by OwaisNizami** [Cnilton/react-native-floating-label-input#122]
Hey there, Your component is great without any doubt.
I just had a little problem adjusting the position of LABEL when using multiline. It actually aligns it to the center of the text box. For example, if I am using multiline with the number of lines(10) then the label in the center looks kind of odd. I want to adjust it to a little below of top left corner where the input will start so that it may slide a little bit instead of moving from center to top.
Can you guide me on how I can do that? already using textAlignVertical="top" to adjust the main text. How to adjust the label position?

**Comment by Cnilton** [comment]
Hi @OwaisNizami, what version are you using? Also, there is a `staticLabel` prop, maybe it is useful for you multiline input. But if it is not what you are looking for, you could set topFocused and topBlurred in `customLabelStyles` prop.

**Comment by OwaisNizami** [comment]
![image](https://user-images.githubusercontent.com/21118326/167348546-9321a7a9-0fbe-4d2d-ab7b-3b32abd85008.jpg)
@Cnilton 
Please check the image. I want to move the 'About You' Text label to the top left corner and I don't want to use the Static label property as I still want to have an animation effect. Please check attached image.

**Comment by Parth8733** [comment]
Okay, 
Thank you for the reply.
  <FloatingLabelInput
                        label="Event Description"
                        hint="Description"
                        textAlignVertical="top"
                        numberOfLines={10}
                        multiline=true
                        maxLength={150}
                        showCountdown=true
                      />
 I am getting error for multiline and showCountdown.

**Comment by Parth8733** [comment]
Thank you @Cnilton

**Comment by OwaisNizami** [comment]
Yes, I had already tried CustomLabel styles. It doesn't work as per my requirement. Please check the image again which I have attached above in the comments. I want to change the initial position of the 'About" label in the multiline input and place it almost in the upper left corner instead of the center of the container and have the little animation from that position then instead of going all the way up from the center. Hope this time I am able to explain What I am trying to ask here. 
@Cnilton 

I have used customLabel like this : 
customLabelStyles={{
          fontSizeFocused: moderateScale(12),
          leftFocused: 20,
          topFocused: 20,
        }}

**Comment by Parth8733** [comment]
hi @OwaisNizami,
if remove topFocused:20 and Add topBlurred: -30. it will move the label near the top. But, I am having a problem with the input text overwriting the Label.

**Comment by OwaisNizami** [comment]
@Parth8733 
I have tried all such scenarios, it's just didn't work the way we want. Let's wait for @Cnilton response. He may guide better about this

**Comment by OwaisNizami** [comment]
hey @Cnilton , any update, please?

**Comment by Cnilton** [comment]
Hi @OwaisNizami, I'm sorry for the holdup, I'll try to check it out tomorrow.

**Comment by hieunm96dev** [comment]
Try using containerStyles={{flex:1 ,minHeight: 50}} with multiline

**Comment by OwaisNizami** [comment]
@Cnilton 
Still waiting for your reply brother

**Comment by askynet** [comment]
For me, it works by setting `topBlurred: -50` i.e negative number
![Screenshot 2022-06-02 at 8 15 51 PM](https://user-images.githubusercontent.com/38763681/171656760-9db890a3-1f3b-4917-b12f-c690c171d761.png)

![Simulator Screen Shot - iPhone 12 Pro Max - 2022-06-02 at 20 15 00](https://user-images.githubusercontent.com/38763681/171656654-3f207166-b8a9-428b-b689-5b2b3dfb38d7.png)
![Simulator Screen Shot - iPhone 12 Pro Max - 2022-06-02 at 20 15 09](https://user-images.githubusercontent.com/38763681/171656709-24e3c84a-a616-4a62-b020-1a22f10e2f23.png)

**Comment by Cnilton** [comment]
@OwaisNizami, I'm really sorry for taking so long... There is a workaround similar to @askynet code. By setting topBlurred and topFocused in customLabelStyles and setting textAlignVertical: 'top' in inputStyles.

**Comment by stereodenis** [comment]
is it connected with this issue?
Label bouncing while adding new line
![Simulator Screen Recording - iPhone 13 - 2022-06-07 at 15 40 53](https://user-images.githubusercontent.com/1406461/172381277-1fda1711-4df3-4a1b-a953-222efa2779b2.gif)

**Comment by Cnilton** [comment]
@stereodenis I think it is related, yes. A workaround for your issue is to set numberOfLines property. I know that this isn't the best, but since the height of the component is updated the animation occurs by calculating the addition of height. If I just calculate once, the label will be stuck at the initial height and eventually will be in front of the input text. 

@stereodenis I'll try to find a solution to this issue. @OwaisNizami did you try out the workaround suggested?

**Comment by Cnilton** [comment]
@stereodenis and @OwaisNizami any updates?

**Comment by stereodenis** [comment]
@Cnilton  I made my own custom input with floating label
